user-access-handlers/src/test/java/no/unit/nva/handlers/HandlerWithEventualConsistencyTest.java

```diff
@@ -0,0 +1,75 @@
+package no.unit.nva.handlers;
```

Comment:
Deleting AddRoleHandlerTest removed test coverage of HandlerWithEventualConsistency so I had to implement new tests for it. Otherwise not related to changeset